src/Umbraco.Web.UI.Client/src/views/propertyeditors/slider/slider.controller.js

```diff
@@ -42,9 +119,10 @@
         var sliderVal = null;
 
         //configure the model value based on if range is enabled or not
-        if ($scope.model.config.enableRange === "1") {
+        if ($scope.model.config.enableRange == true) {
             //If no value saved yet - then use default value
-            if (!$scope.model.value) {
+            //If it contains a single value - then also create a new array value
+            if (!$scope.model.value || $scope.model.value.indexOf(",") == -1) {

```

Comment:
This change was important.
If you had a single-value slider (and had a saved value) and then changed it to a range-slider afterwards, then the range-slider was broken, because model.value contains a value, but it could not convert it into an array.